spec/factories/users.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :user do
-    client_slug { ENV['CLIENT_SLUG_DEFAULT'] || 'ncr' }
+    client_slug { ENV['CLIENT_SLUG_DEFAULT'] || nil }
```

Comment:
fair point. Removing the env var required a bit more clean up in the `Proposal` factory as well, since it was also referenced there.